packages/create-gatsby/src/index.ts

```diff
@@ -44,6 +45,42 @@ const questions = [
   },
 ]
 
+const supportedOptionTypes = [`string`, `boolean`, `number`]
+
+const makePluginConfigQuestions = (
+  selectedPlugins: Array<string>
+): Array<any> => {
+  const formPrompts = selectedPlugins.map((pluginName: string) => {
+    const schema = pluginSchemas[pluginName]
+    const { keys: options } = schema
```

Comment:
The plugins aren't all objects, and don't all have a `keys` prop, so we should check for this at runtime. This will also narrow the type for TypeScript.
```suggestion
    if (typeof schema === `string` || !(`keys` in schema)) {
      return []
    }
    const { keys: options } = schema
```